src/lua/fiber.c

```diff
@@ -156,11 +156,10 @@ static struct fiber *
 lbox_checkfiber(struct lua_State *L, int index)
 {
 	uint64_t fid;
-	if (lua_type(L, index) == LUA_TNUMBER) {
+	if (lua_type(L, index) == LUA_TNUMBER)
 		fid = luaL_touint64(L, index);
-	} else {
+	else
 		fid = *(uint64_t *)luaL_checkudata(L, index, fiberlib_name);
-	}
```

Comment:
This codestyle fix doesn't seem to be related to the patch

Reply:
Remove codestyle fix